instrumentation/metric/histogram.go

```diff
@@ -16,24 +16,24 @@ type Histogram struct {
 
 type histogramExport struct {
 	Name    string
-	Min     int64
-	P50     int64
-	P95     int64
-	P99     int64
-	Max     int64
+	Min     float64
```

Comment:
The conversion to millis should happen only in `LogRow`

Reply:
I disagree because these values are going to be exported through http api. They should be uniform.